python-sdk/tests/test_examples.py

```
@@ -237,9 +237,10 @@ def uses_colbert_service(file_path: str) -> bool:
                     pytest.fail(f"Error running main function in {example_file}: {e!s}")
         trace.send_spans()
         if parity_prefix:
-            # In parity-check mode, record trace ID directly (avoids share API call
-            # which may fail if the trace hasn't been ingested yet)
             trace_urls[example_file] = trace.trace_id or "unknown"
         else:
-            trace_urls[example_file] = trace.share()
+            try:
+                trace_urls[example_file] = trace.share()
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)